src/admin-views/tribe-options-licenses.php

```diff
@@ -36,6 +36,9 @@
 	$html .= "<p> $network_all_sites_text $network_admin_only </p>";
 }
 
+// Explanatory text about license settings for the tab information box
+$support_html = __( '<p>The details of your calendar plugin and settings are often needed for you or our staff to help troubleshoot an issue. Please opt-in below to automatically share your system information with our support team. This will allow us to assist you faster if you post in our forums <a href="%1$s" target="_blank">forums</a>. You can see exactly what information you\'ll be sharing by viewing the System Info section on the <a href="%2$s" target="_blank">Help Tab</a>.</p>', 'tribe-common' );
```

Comment:
Wonder if we could minimize the use of HTML in the translated text here? The opening and closing `<p>` tags could definitely be moved out, optionally we could also replace the anchor tags etc with numbered placeholders.